CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [v0.5.1](https://github.com/Nexus-Mods/NexusMods.App/releases/tag/v0.5.1) - 2024-06-03
+
+This is a minor release containing bugfixes and small UI improvements.
+
+### Fixes
+
+- Fixed the "View In Library"-button attempting to opening deleted Loadouts ([#1499](https://github.com/Nexus-Mods/NexusMods.App/pull/1499)).
```

Comment:
This has not been back ported to 0.5 branch.
I can do a PR if you want but halgari said to skip it for 0.5